development/webpack/webpack.prod.config.js

```
@@ -1,41 +1,64 @@
 const TerserPlugin = require('terser-webpack-plugin');
+const path = require('path');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)